ds-card-reader/device/physical.go

```diff
@@ -90,12 +91,18 @@ func (reader *CardReaderPhysical) Status() error {
 func (reader *CardReaderPhysical) Write(commandName string, cardNumber string) {
 	// assemble the values that will be propagated throughout the
 	// device service
+	commandvalue, err := dsModels.NewCommandValueWithOrigin(
+		commandName,
+		edgexcommon.ValueTypeString,
+		cardNumber,
+		time.Now().UnixNano()/int64(time.Millisecond),
+	)
+	if err != nil {
+		fmt.Errorf("error on NewCommandValueWithOrigin for %v", commandName)
```

Comment:
what are we trying to do with this err formatting?  should we log it here?
Also should we include err details in part of Errorf?